tests/ClientTest.php

```diff
@@ -1838,25 +1828,6 @@ public function test_server_variable()
         }
     }
 
-    public function testEncodeTooDepth()
```

Comment:
I deleted the entire method because we do not need to test INTERNAL PHP FUNCTIONS!